config/sources/families/include/rockchip64_common.inc

```diff
@@ -145,7 +165,7 @@ uboot_custom_postprocess()
 		:
 	elif [[ $BOOT_SCENARIO == "spl-blobs" ]]; then
 
-		tools/mkimage -n $BOOT_SOC -T rksd -d $RKBIN_DIR/$DDR_BLOB:spl/u-boot-spl.bin idbloader.img
+		tools/mkimage -n $BOOT_SOC_NAME -T rksd -d $RKBIN_DIR/$DDR_BLOB:spl/u-boot-spl.bin idbloader.img
```

Comment:
Please rename it back to $BOOT_SOC, let's not introduce additional variables.

Reply:
I will change this after checking how the override will work.